worldedit_commands/init.lua

```diff
@@ -643,6 +701,13 @@ minetest.register_chatcommand("/spiral", {
 		worldedit.player_notify(name, count .. " nodes added")
 	end,
 	function(name, param)
+		if nil ~= area_protection.areas and not minetest.check_player_privs(name, {areas = true}) then

```

Comment:
These constructs will only stay if there are shapes that cannot be resolved in a reasonable manner.